pixeltable/functions/huggingface.py

```diff
@@ -85,6 +79,28 @@ def clip_image(image: Batch[PIL.Image.Image], *, model_id: str) -> Batch[np.ndar
     return [embeddings[i] for i in range(embeddings.shape[0])]
 
 
+@pxt.udf(batch_size=32)
+def detr_detections(image: Batch[PIL.Image.Image], *, model_id: str, threshold: float = 0.5) -> Batch[dict]:
+    env.Env.get().require_package('transformers')
+    from transformers import DetrImageProcessor, DetrForObjectDetection
+
+    model = _lookup_model(model_id, lambda x: DetrForObjectDetection.from_pretrained(x, revision='no_timm'))
+    processor = _lookup_processor(model_id, lambda x: DetrImageProcessor.from_pretrained(x, revision='no_timm'))
+
+    inputs = processor(images=image, return_tensors='pt')
+    outputs = model(**inputs)
+
+    results = processor.post_process_object_detection(outputs, threshold=threshold)
+    return [
+        {
+            'scores': [score.item() for score in result['scores']],
+            'labels': [model.config.id2label[label.item()] for label in result['labels']],
```

Comment:
Wondering if we should also include the ids, might be easier to work with in some scenarios.